src/styles.css

```diff
@@ -78,7 +78,8 @@ summary,
 time,
 mark,
 audio,
-video {
+video,
```

Comment:
I wouldn't add my styling to the base elements. You should define a class like `search-input` or `pokedex-input` and style that. 

If we attach this specific styling to each default element, what happens if we need an default HTML `input` or `select`? In general, always try to build on top of something (extensibility) versus a tightly coupled change like this.